### PR TITLE
Allow importing of individual files

### PR DESCRIPTION
**Pull request by zachmullen**

### CODE REVIEW

**Comment by zachmullen** [comment]
@cpatrick PTAL, this will better facilitate synchronization of a local filesystem to a girder hierarchy.

File: girder/utility/filesystem_assetstore_adapter.py [hunk 1]
```diff
@@ -332,19 +332,39 @@ def _importDataAsItem(self, name, user, folder, path, files,
             self.importFile(item, os.path.join(path, fname),
                             user, name=fname)
 
+    def _hasOnlyFiles(self, path, files):
+        return all(map(os.path.isfile, (os.path.join(path, fname)
```
Comment (cpatrick) [inline girder/utility/filesystem_assetstore_adapter.py]: nit: Since this is a comprehension in a map in an all, can we have a small `#` comment. :smile:

**Comment by cpatrick** [comment]
Other than the one nit, LGTM.

File: girder/utility/filesystem_assetstore_adapter.py (hunk 1 above)
Comment (fogleman) [inline girder/utility/filesystem_assetstore_adapter.py]: IMO, just get rid of the map:

    return all(os.path.isfile(os.path.join(path, fname)) for fname in files)

**Pull request merged**